esx_service/vmdk_ops.py

```diff
@@ -478,6 +478,7 @@ def get_vol_path(datastore, tenant_name=None):
     # volumes is created on <datastore>/DOCK_VOLS_DIR
     # If the command is running under a tenant, the foler for Dock volume
     # is created on <datastore>/DOCK_VOLS_DIR/tenant_name
+    dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
     if tenant_name:
         path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR, tenant_name)
```

Comment:
os.path.join(dock_vol_path, tenant_name). I am okay if you decide to leave it as-is.